prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.ts

```diff
@@ -114,8 +112,15 @@ export class EnrolleeOverviewComponent extends AdjudicationContainerComponent im
         exhaustMap((enrolleeId: number) => this.adjudicationResource.getPlrInfoByEnrolleeId(enrolleeId)
           .pipe(
             map((plrInfo: PlrInfo[]) => this.plrInfo = plrInfo),
-            catchError(_ => of([]))))
-      ).subscribe();
+            catchError(_ => of([])))),
+        exhaustMap(() =>
+          this.isPaperEnrollee(this.enrollee) ?
+            this.paperEnrolmentResource.getAdjudicationDocuments(+this.route.snapshot.params.id) :
+            of(null))
+      ).subscribe(
+        // Note that these documents are currently not displayed for a Completed Paper Submission
+        // due to how `showAdjudication` is set and causing DocumentAttachmentsComponent to be hidden
+        documents => this.documents = documents);
```

Comment:
Missing the typing on documents. As well as what looks like some formatting

Reply:
What's the formatting issue(s)?

Reply:
documents is not typed
``` subscribe((documens: type) => this.documents = documents) ```
Also not sure what the comment represents in this case, maybe jsut add a TODO: display documents for completed paper submission

Reply:
I removed comment and created a Jira ticket instead:  PRIME-2038